### PR TITLE
Added documentation to input component

### PR DESCRIPTION
**Pull request by vRuslan**
Closes IBM/carbon-components-angular#

Added documentation to input component 

#### Changelog

**New**

* 

**Changed**

* src/input/input.directive.ts

**Removed**

*

### CODE REVIEW

File: src/input/input.directive.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import { Directive, HostBinding } from "@angular/core";
-
+/**
+ * A convinence directive for applying styling to a input element.
```
Comment (magedhennawy) [inline src/input/input.directive.ts]: `A convinence directive` is not necessary. 
Maybe replace that with `A component`

File: src/input/input.directive.ts [hunk 2]
```diff
@@ -1,5 +1,15 @@
 import { Directive, HostBinding } from "@angular/core";
-
```
Comment (magedhennawy) [inline src/input/input.directive.ts]: Can we have an empty line here?

File: src/input/input.directive.ts (hunk 1 above)
Comment (magedhennawy) [inline src/input/input.directive.ts]: There's a typo in all documentation that has `A convinence directive`. 

I will accept this and change the rest after it goes through

Comment (vRuslan) [reply src/input/input.directive.ts]: Ok I'll change

File: src/input/input.directive.ts [hunk 3]
```diff
@@ -1,5 +1,16 @@
 import { Directive, HostBinding } from "@angular/core";
 
+/**
+ * A Component for applying styling to a input element.
```
Comment (zvonimirfras) [inline src/input/input.directive.ts]: Should be `A directive for applying styling to an input element.`

**Pull request merged**